scroll-animations-1/Overview.bs

```diff
@@ -517,7 +521,7 @@ spec: cssom-view-1; type: dfn;
 
 	<pre class='propdef shorthand'>
 	Name: view-timeline
-	Value: [ <<'view-timeline-name'>> || <<'view-timeline-axis'>> ]#
+	Value: [ <<single-timeline-name>> || <<single-timeline-axis>> ]#
```

Comment:
Or `<single-timeline>#` with `<single-timeline> = <single-timeline-name> || <single-timeline-axis>` if you want to align with animations/background/etc.